src/webview/memory-webview-view.tsx

```diff
@@ -137,9 +136,11 @@ class App extends React.Component<{}, MemoryAppState> {
             this.setState({
                 decorations: decorationService.decorations,
                 memory: this.convertMemory(response),
-                memoryReference: completeOptions.memoryReference,
-                offset: completeOptions.offset,
-                count: completeOptions.count,
+                memoryReadArguments: completeOptions,
+                userReadArguments: {
+                    ...this.state.userReadArguments,
+                    memoryReference: completeOptions.memoryReference
```

Comment:
In principle, the `memoryReference` should come from the configured `userReadMemoryArguments`: are there cases where that won't be the case?

Reply:
Yes, the only case is, when it comes from the outside (e.g., if you open a memory). I forgot to update the `setOptions` method.